zerver/openapi/zulip.yaml

```
@@ -9105,6 +9117,31 @@ paths:
           schema:
             type: boolean
           example: true
+        - name: email_address_visibility
+          in: query
+          description: |
+            The [policy][permission-level] this user has selected for which other
+            users in this organization can see the real email address of this user.
+
+            - 1 = everyone
+            - 2 = members only
+            - 3 = administrators only
+            - 4 = nobody
+            - 5 = moderators only
```

nit: I think title-case (like we have in the doc for the old realm setting) is a bit more readable in this sort of table:

```suggestion
            - 1 = Everyone
            - 2 = Members only
            - 3 = Administrators only
            - 4 = Nobody
            - 5 = Moderators only
```

(Similarly elsewhere.)

---

Updated for this. We can make this consistent at other places too, we still use lower cases in some places.

---

I think it's not a big priority to fix that, in that we intended to replace almost all of these settings with groups, but maybe it's worth a sweep since it's not hard and we will still mention there in **changes** API documentation about previous states.